src/types.d.ts

```diff
@@ -11,3 +11,29 @@ type MeasurementRequest = {
 interface CommandInterface<OPT> {
 	run(socket: Socket, measurementId: string, testId: string, options: OPT): Promise<void>;
 }
+
+type Probe = {
+	location: ProbeLocation;
+};
+
+type ProbeLocation = {
+	continent: string;
+	region: string;
+	country: string;
+	city: string;
+	asn: string;
+	latitude: string;
+	longitude: string;
+	state: string | undefined;
+};
+
+type WsApiError = {
+	message: string;
+	info: {
+		code?: string;
```

Comment:
`socketId` is missing? If we don't use it on the probe side, we can remove it on the API side as well because I don't really think it's useful there as well.

Reply:
it might be useful for debugging purposes. e.g. user gets disconnected, reports us the issue - the more data they give us, the better.

Reply:
Where user will get it? Do we show this id somewhere in the probe logs?

Reply:
You got me. We don't.